pallets/runtime/tests/src/asset_test.rs

```diff
@@ -216,154 +216,15 @@ fn non_issuers_cant_create_tokens() {
     });
 }
 
-fn valid_custodian_allowance() {
+#[test]
+fn valid_transfers_pass() {
```

Comment:
I suspect more tests are needed for branch coverage.

Reply:
Yeah, there are no tests around the new functionality right now. They are in my todo list.